cumulus/src/designflow/pnr.py

```diff
@@ -65,7 +65,8 @@ def doTask ( self ):
         else:
             print( 'PnR.doTask() run in interactive CGT mode.' )
             PnR.textMode = False
-        from .. import Etesian, Anabatic, Katana, Bora, Tramontana, Tutorial, Viewer, Unicorn
+       #from .. import Etesian, Anabatic, Katana, Bora, Tramontana, Tutorial, Viewer, Unicorn
```

Comment:
Remove commented line

Reply:
The comment is only temporary, it's the uncommented line that will go away as soon as I add support for ``tramontana`` and ``tutorial`` (they are not deprecated tools, but new one and keep it one).

Reply:
mmm, surely you can just add that when it's done?